src/components/favorites-tab.less

```diff
@@ -7,35 +7,7 @@
 }
 
 .FavoritesTab-actions {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  gap: 5px;
-}
-
-.FavoritesTab-action {
-  text-decoration: none;
-  display: flex;
-  width: 22px;
-  height: 22px;
-  align-items: center;
-  justify-content: center;
-
-  &:hover,
-  &:active {
-    .m-box-shadow-dark;
-    text-decoration: none;
-    background-color: @edu-white;
-  }
-
-  &.FavoritesTab-action--delete {
-    color: @edu-error-color;
-
-    &:hover,
-    &:active {
-      color: @edu-error-color;
-    }
-  }
```

Comment:
Hadn't noticed there was this amount of duplication between these action buttons. Nice that you reduced it 👍